app/models/subscription_target.rb

```diff
@@ -0,0 +1,18 @@
+class SubscriptionTarget
+  include AuthenticatableApiEntity
+
+  attr_accessor :target_type,
+                :chapter_short_code,
+                :goods_nomenclature_item_id,
+                :hierarchical_description,
+                :meta
+
+  def self.all(token, params)
+    return nil if token.nil? && !Rails.env.development?
+
+    set_collection_path "/uk/user/subscription_targets/#{token}"
```

Comment:
I think this could be a problem as set_collection_path is a class method. 
Better to call the collection method from here with the path you've created.